internal/core/src/query/visitors/ExecExprVisitor.cpp

```diff
@@ -306,12 +323,68 @@ ExecExprVisitor::ExecRangeVisitorImpl(FieldId field_id,
         }
         results.emplace_back(std::move(chunk_res));
     }
+
     auto final_result = AssembleChunk(results);
     AssertInfo(final_result.size() == row_count_,
                "[ExecExprVisitor]Final result size not equal to row count");
     return final_result;
 }
 
+template <typename T, typename InRangeFunc>
+std::enable_if_t<ExecExprVisitor::IsAllowedType<T>::value, bool>
+ExecExprVisitor::InChunkRange(InRangeFunc inRange,
+                              segcore::FieldChunkMetrics& fieldChunkMetrics,
```

Comment:
this interface is a little redundant, T is datatype , and third argument is also datatype.